src/connector/src/common.rs

```diff
@@ -310,6 +310,8 @@ pub struct ClickHouseCommon {
     pub database: String,
     #[serde(rename = "clickhouse.table")]
     pub table: String,
+    #[serde(rename = "clickhouse.sign")]
```

Comment:
Can this field be read from the clickhouse system table instead of passed from user?

Reply:
We can't do it. Because users can specify the name of the `sign` in `ClickHouse`, we don't know which column is the `sign`, so we need the user to tell us.

Reply:
Just like we can query the storage engine type of the clickhouse table. `sign` field is also specified when user creates a clickhouse table and it's a metadata similar to the engine type and I think there should be a system table to store it.

Reply:
okk